contributing-guidelines/template.md

```diff
@@ -362,7 +362,7 @@ And they'll render like this:
 [button links](../docs/core/index.md)
 ```
 
-You can see an example of buttons in action at the [Intune docs](https://learn.microsoft.com/intune/get-started/choose-how-to-enroll-devices).
+You can see an example of buttons in action in the [Intune documentation](https://learn.microsoft.com/intune/get-started/choose-how-to-enroll-devices).
```

Comment:
Link does not work, adding to broken link tab. Perhaps try https://learn.microsoft.com/mem/intune/enrollment/